test/numerical/TestConv.cpp

```diff
@@ -156,6 +160,12 @@ int main(int argc, char *argv[]) {
   assert(isOMConvTheSameAsNaiveImplFor(
              1, 512, 1000, 13, 13, 1, 1, 0, 0, 0, 0, ConvAutoPad::NOTSET) &&
          "failed test from test_squeezenet_cpu");
+  assert(isOMConvTheSameAsNaiveImplFor(
+             3, 64, 64, 55, 55, 1, 1, 1, 1, 1, 1, ConvAutoPad::NOTSET) &&
```

Comment:
Looks like the intention here is for conv ops that can be expressed by matmul, while the two new added test are not.

Reply:
Changed the condition. Thank you.